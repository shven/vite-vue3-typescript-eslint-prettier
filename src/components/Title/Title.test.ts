import { mount } from '@vue/test-utils';
import Component from './VueTitle.vue';

describe('VueTitle.vue', () => {
    const wrapper = mount(Component, {
        props: {
            label: 'Title goes here',
            level: 'h2',
            size: 'medium'
        }
    });

    it('Match snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
