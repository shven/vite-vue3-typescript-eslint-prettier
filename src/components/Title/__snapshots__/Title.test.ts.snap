// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VueTitle.vue > Match snapshot 1`] = `
<h2
  class="title title--medium"
  data-v-a0ca2fca=""
>
  Title goes here
</h2>
`;
